ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -82,6 +82,8 @@
     reason="Tests are not running on AWS deployed cluster"
 )
 
+skip_always = pytest.mark.skipif(True, reason="Test was marked for intentional skipping")
```

Comment:
please try passing the reason from outside

Reply:
For now, I replaced this with a constant message saved in `ERRATIC_TIMEOUTS_SKIP_REASON`